scss/bootstrap/overrides/_variables.scss

```diff
@@ -22,11 +22,8 @@ $crm-page-topbar-bg:           #222831 !default;
 $crm-page-topbar-bg-light:     #393E46;
 $crm-page-topbar-color:        #fff !default;
 $crm-page-topbar-item-padding: 15px !default;
-$crm-accordion-header-color:   #f3f6f7;
-$crm-dasboard-header:          #4D4D69;
-$crm-h3-color:                 #525966;
-$crm-wizard-border:            #eff3f5;
-$crm-text-muted:               #9494A5;
+
+$crm-placeholder:              #9494A5;
```

Comment:
is this color part of the palette? Also, can you please give it a name like `$crm-{name_of_color}` which you then assign to `crm-placeholder`?

btw, are we sure about this name? Why can't we just use `$crm-{name_of_color}`? What if we want to use this color with something that is not a placeholder?